content/en/tracing/profiler/profiler_troubleshooting.md

```diff
@@ -61,6 +61,20 @@ jdk.ObjectAllocationOutsideTLAB#enabled=true
 
 [Learn how to use override templates.](#creating-and-using-a-jfr-template-override-file)
 
+## Enabling the heap profiler
+
+To enable the heap profiler, start your application with the `-Ddd.profiling.heap.enabled=true` JVM setting or the `DD_PROFILING_HEAP_ENABLED=true` environment variable.
+
+Alternatively, you can enable the following events in your `jfp` [override template file](#creating-and-using-a-jfr-template-override-file):
+
+```
+jdk.OldObjectSample#enabled=true
+```
+
+There is no support for reference chains at this time. Please [open a support ticket][2] if it's something you are interested in.
```

Comment:
I am not sure we should be mentioning 'reference chains' as it is a rather implementation specific term and most of the users would not have any idea what it is.

Reply:
I would still want to gather a pulse from customers whether they are interested in the feature. @GaurabAryal where do you think is the best place to gather feedback and interest on it?

Reply:
I added a suggested rewording. Let me know what you think.